src/helpers/i18n-utils.js

```diff
@@ -64,16 +64,16 @@ export const getDirectionProperty = dir => dir === 'rtl' ? 'right' : 'left';
 
 export const getLangPropertyDirection = language => getDirectionProperty(getLanguageDirection(language));
 
-export const getLanguageLocale = language => LANGUAGES[language].locale;
+export const getLanguageLocale = language => (LANGUAGES[language] || LANGUAGES[DEFAULT_UI_LANGUAGE]).locale;
```

Comment:
This does not make sense... you want to tell me language is not in LANGUAGES?

All languages should be there... https://github.com/Bnei-Baruch/kmedia-mdb/blob/2c9177def5010315abd3a37d6e80616341439f3f/src/helpers/consts.js#L187


The only issue if language is "broken" or wrong... what do you have in this case?